theme/brb/views/dataprivacy/privacy.hbs

```diff
@@ -0,0 +1,113 @@
+<div class="section-imprint {{#unless loggedin}}loggedout{{/unless}}">
+	{{#unless loggedin}}
+		<h4>Datenschutzerklärung</h4>
+	{{/unless}}
+    <p>Hinweise zum Datenschutz entnehmen Sie bitte folgender Seite.</p>
+	<h5>
+		<strong>Privatsph&auml;re und Datenschutz</strong>
+	</h5>
+	<p>
+		Mit der Schul-Cloud Brandenburg eröffnet das Hasso-Plattner-Institut (HPI) die Möglichkeit, diverse Online-Lernangebote über eine Weboberfläche zu nutzen und fördert ein vernetztes Lernen und Arbeiten auf allen Ebenen der Schule. Alle Nutzer/innen – Schüler/innen bzw. Lehrkräfte – erhalten einen eigenen Zugang zur Schul-Cloud Brandenburg. Über diesen können die Schüler/innen die bereitgestellten Arbeitsmaterialien und Aufgaben bearbeiten und weitere digitale Lernmaterialien auf schlanken Endgeräten nutzen. Darüber hinaus werden über die Schul-Cloud Brandenburg verschiedene Anwendungen, wie ein Kalender bzw. Stundenplan sowie Werkzeuge für die Kommunikation und Kollaboration bereitgestellt. Die Schul-Cloud Brandenburg bietet zudem nicht nur eine sichere Dateiablage für alle Nutzer/innen, Gruppen, Klassen und Schulen, sondern auch geschützte Arbeits- und Informationsbereiche.</p>
+	<p>
+		Zur Nutzung der Schul-Cloud Brandenburg ist die Erhebung und Speicherung personenbezogener Daten unvermeidlich. Ihrer Anmeldung gehen die Informationen über die Art, den Zweck, die Verarbeitung und Nutzung der erhobenen Daten sowie die Zustimmung zu den Nutzungsbedingungen voraus. Der Schutz Ihrer Daten sowie die rechtmäßige Erhebung, Verarbeitung und Nutzung ist uns besonders wichtig.</p>
+	<h5>
+		<strong>1. Verantwortliche Stelle</strong>
+	</h5>
+	<p>
+		Verantwortliche Stelle für Verarbeitung Ihrer personenbezogenen Daten ist die Schule, der Sie angehören oder der Schulträger. Ihre Schule oder der Schulträger ist beauftragende Stelle und obere Aufsichtsbehörde.</p>
+	<h5>
+		<strong>2. Erhebung personenbezogener Daten</strong>
+	</h5>
+	<p>
+		Gem. Art. 4 Nr. 1 der europäischen Datenschutz-Grundverordnung (DS-GVO) beschreiben personenbezogene Daten Einzelangaben über persönliche oder sachliche Verhältnisse einer bestimmten oder – ohne besondere Schwierigkeiten bestimmbaren – natürlichen Person (Betroffene/r). Zu diesen Informationen gehören vor allem Details, die Rückschlüsse auf Ihre Identität zulassen, zum Beispiel Ihren Namen oder Ihre Email-Adresse. Statistische Daten, die keine Rückschlüsse auf Ihre Person zulassen, sind keine personenbezogenen Daten.</p>
```

Comment:
please run a spelling checker 
![image](https://user-images.githubusercontent.com/4550014/61455894-0dcf6f00-a965-11e9-8c7c-2541566056d6.png)

Reply:
use E-Mail-Adresse and DSGVO